pkg/v1/providers/tests/clustergen/gencluster_params.py

```diff
@@ -52,7 +52,10 @@ def write_test_cases(params_file, test_dir):
                     if k == "AZURE_CUSTOM_TAGS" and v.startswith('tagKey1='):
                         cfg_args.append('{}: {}'.format(k, 'tagKey1=tagValue1, tagKey2=tagValue2'))
                     elif v != "NA":
-                        cfg_args.append('{}: {}'.format(k, v.replace("<comma>", ",")))
+                        if type(v) == str:
```

Comment:
Can you point to a situation where this is needed? All the pict csv files generated should be a fully formed CSV with
(R+1) x C cells, with quoted strings in the value rows.

Reply:
We think we were running into problems with the vsphere variable, and that was our initial fix. We've since updated our PR to wildcard on the vsphere semver, and we're no longer seeing that issue. We've reverted these lines.

Reply:
You're right, this wasn't needed.